### ISSUE TITLE
go get failure

### ISSUE THREAD

**Issue by chriseaton** [wellington/wellington#175]
Trying to go get wellington fails:
go version go1.5.2 linux/amd64

```
$ go get -u github.com/wellington/wellington
# github.com/wellington/go-libsass/libs
wellington/go-libsass/libs/unity.cpp:37:42: fatal error: ../libsass-build/to_string.cpp: No such file or directory
 #include "../libsass-build/to_string.cpp"
                                          ^
compilation terminated.
```

**Comment by drewwells** [comment]
Sorry, libsass is in the middle of shipping a release and I pushed some untested code to go-libsass. Some changes have broken go-libsass master, issue here: https://github.com/sass/libsass/issues/1896

Can you try again on master? Also, make sure you have vendor experiment on `GO15VENDOREXPERIMENT=1 go install github.com/wellington/wellington/wt`

**Comment by drewwells** [comment]
Please reopen if this is still an issue. Vendor experiment is on by default in 1.6.x and no longer an option in the soon to be released 1.7
